documentation: add info about cmdargs (#1862)
mozilla/mozregression: docs/documentation/usage.md

        string.pref = Hello, world!
        integer.pref = 1

> - Forward arguments to tested binary (note that you must use '=' for arguments beginning with '-' so that the parser doesn't get confused) (Second note: in the configuration file these are called `cmdargs` instead. See [here](configuration.html).

Reviewer: ```suggestion
- Forward arguments to tested binary. Note that you must use '=' for arguments beginning with '-' so that the parser doesn't get confused. Additionally, this is equivalent to `cmdargs` in the [configuration file](configuration.html).
```
